packages/client/src/checkout/index.ts

```diff
@@ -14,7 +14,7 @@ export { default as patchCheckout } from './patchCheckout';
 export { default as patchDeliveryBundleUpgrades } from './patchDeliveryBundleUpgrades';
 export { default as patchCheckoutCompletePayment } from './patchCheckoutCompletePayment';
 export { default as patchGiftMessage } from './patchGiftMessage';
-export { default as getCharges } from './getCharges';
+export { default as getCharges } from './getCheckoutOrderCharge';
```

Comment:
```suggestion
export { default as getCheckoutOrderCharge } from './getCheckoutOrderCharge';
```

Reply:
This may include some additional changes on other files

Reply:
You are right, I need to update the imports on redux actions